app/controllers/orgs/rosters_controller/lti_dependency.rb

```diff
@@ -71,7 +71,14 @@ def lms_membership
 
       begin
         membership_service.students(body_params: current_organization.lti_configuration.context_membership_body_params)
-      rescue Faraday::ClientError, JSON::ParserError
+      rescue Faraday::ClientError, JSON::ParserError => error
```

Comment:
What do you think about moving this error builder/reporter to its own method like we do here? https://github.com/education/classroom/blob/master/app/services/create_github_repo_service.rb#L89

Reply:
True. That is a lot better. I'll make those changes 😄